service/debugger/debugger.go

```diff
@@ -354,6 +344,49 @@ func (d *Debugger) CreateBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoin
 	return createdBp, nil
 }
 
+// SetExecutionPoint will jump the execution point.
+func (d *Debugger) SetExecutionPoint(requestedExecPoint *api.ExecutionPoint) error {
+	d.processMutex.Lock()
+	defer d.processMutex.Unlock()
+
+	if len(requestedExecPoint.File) > 0 {
```

Comment:
I know this is what we are doing for breakpoints, but that's because of backwards compatibility. Given that the intended use of this API is to always call FindLocations first I would lean towards not repeating the sins of the past.